core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -996,50 +1006,52 @@ class DAGScheduler(
             if (failedEpoch.contains(execId) && smt.epoch <= failedEpoch(execId)) {
               logInfo("Ignoring possibly bogus ShuffleMapTask completion from " + execId)
             } else {
-              stage.addOutputLoc(smt.partitionId, status)
+              shuffleStage.addOutputLoc(smt.partitionId, status)
             }
-            if (runningStages.contains(stage) && stage.pendingTasks.isEmpty) {
-              markStageAsFinished(stage)
-              logInfo("looking for newly runnable stages")
+            if (runningStages.contains(shuffleStage) && shuffleStage.pendingTasks.isEmpty) {
+              markStageAsFinished(shuffleStage)
+              logInfo("looking for newly runnable shuffleStages")
```

Comment:
I think that this log message should remain unchanged, since technically I think we'll consider scheduling of all runnable stages, which might also include result stages